docs/guide/apm-getting-started.asciidoc

```diff
@@ -25,9 +25,9 @@ It also makes it much easier to keep an eye on how many times specific errors ha
 === APM Components
 Elastic APM consists of four components:
 
-* Elasticsearch
-* APM agents
-* APM Server
+* {ref}/index.html[Elasticsearch]
+* {apm-agents-ref}/index.html[APM agents]
```

Comment:
I can't see where this is defined? If we need this I guess it should be added to https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc

Reply:
yes, I haven't pushed those changes yet

Reply:
https://github.com/elastic/docs/pull/237